modules/auth/assembly-enabling-authentication.adoc

```diff
@@ -0,0 +1,34 @@
+[id='enabling-authentication']
+= Enabling authentication in {product}
+
+Authentication within {product} facilitates user sign-in, identification, and access to external resources.
+It supports multiple authentication providers.
+
+Authentication providers are typically used in the following ways:
+
+- One provider for sign-in and identification.
+- Additional providers for accessing external resources.
+
+The {product} supports the following authentication providers:
+
+Auth0:: `auth0`
+Atlassian:: `atlassian`
+Microsoft Azure:: `microsoft`
+Microsoft Azure Easy Auth:: `azure-easyauth`
+Bitbucket:: `bitbucket`
+Bitbucket Server:: `bitbucketServer`
+Cloudflare Access:: `cfaccess`
+GitHub:: `github`
+GitLab:: `gitlab`
+Google:: `google`
+Google IAP:: `gcp-iap`
+OIDC:: `oidc`
+Okta:: `okta`
+OAuth 2 Custom Proxy:: `oauth2Proxy`
+OneLogin:: `onelogin`
+SAML:: `saml`
+
+For each provider, follow the dedicated procedure to:
```

Comment:
```suggestion
For each provider that you want to use, follow the dedicated procedure to:
```

Reply:
Just to clarify that they don't actually have to follow the procedure for each provider, unless they want to.